src/Common/src/Internal/Runtime/EEType.Constants.cs

```diff
@@ -101,4 +101,96 @@ internal enum EETypeKind : ushort
         /// </summary>
         GenericTypeDefEEType = 0x0003,
     }
+
+    /// <summary>
+    /// These are flag values that are rarely set for types. If any of them are set then an optional field will
+    /// be associated with the EEType to represent them.
+    /// </summary>
+    [Flags]
+    internal enum EETypeRareFlags : int
```

Comment:
Could you please delete `enum RareFlags` from Runtime.Base\src\System\Runtime\EEType.cs and switch code to use this enum?